### DISCUSSION TITLE
Extend menu not working when holding button and power on

### DISCUSSION THREAD

**Discussion by kevtheskin** [egzumer/uv-k5-firmware-custom#365] [Q&A]
Hello there, The  Extend menu not working when holding button below Ptt and power on .  Latest firmware. Can this be fixed?

**Comment by egzumer** [comment]
It does work, if you cant enter it it is a problem with your radio

**Comment by ErikS-web** [comment]
There are 2 buttons below the PTT button...

![Side-buttons](https://github.com/egzumer/uv-k5-firmware-custom/assets/148579604/9a804159-8202-4fa5-8143-3026144714b1)

Press the top 'FN1' and 'PTT' simultaneously and then _switch on the radio_ and the keep keys pressed...

Wait until text on screen "Release all keys"
